src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const result = {};
```

Comment:
please rename the variable to give more context ([checklist](https://github.com/mate-academy/js_inverse_robot/blob/master/checklist.md) - point 3)